src/Application/ConfigResolver.php

```diff
@@ -209,4 +205,30 @@ private static function preparePreset(array $preset, array $config): array
 
         return $preset;
     }
+
+    private static function resolvePreset(string $testPreset, Composer $composer): string
+    {
+        if ($testPreset === '' || $testPreset === 'default') {
+            return self::DEFAULT_PRESET;
```

Comment:
If I understand, when there is no 'preset' in `phpinsights`, we return the default preset ? 

So we can't guess it from composer, right ?

Reply:
We could guess it from composer, but that would be a breaking change. At least that was my understanding. I was trying to add this functionality without making any breaking changes.